CONTRIBUTING.md

```diff
@@ -223,80 +221,6 @@ Follow the guidelines for setting up a development environment, verifying the ch
 and submitting a pull request, as described in the [main section about updating an
 exercise test suite](#updating-an-exercise-test-suite).
 
-## Porting an Exercise to Another Language Track
```

Comment:
This is also not a straightforward replacement.
There is information here that is not present in https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/porting-an-exercise.md